spec/helpers/comments_helper_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+# Specs in this file have access to a helper object that includes
+# the CommentsHelper. For example:
+#
+# describe CommentsHelper do
+#   describe "string concat" do
+#     it "concats two strings with spaces" do
+#       expect(helper.concat_strings("this","that")).to eq("this that")
+#     end
+#   end
+# end
+RSpec.describe CommentsHelper, type: :helper do
+  pending "add some examples to (or delete) #{__FILE__}"
+end
```

Comment:
- [ ] You seem to have generate your files using scaffold command, else this file would not have been present. You were asked not use scaffold command so that you will have a practical understanding of how Rails routing works. Kindly remove this file.

Reply:
I definitely didn't use a scaffold. I don't know why these files are there. Maybe it is because I was experimenting with rails last week to understand it. I used a lot of auto generate features at the time. I tried to remove the files auto generated after experimenting with them. I guess these file escaped my notice.

Reply:
Okay. But you still need to remove the indicated files, since they are auto-generated as auto-generated files are disallowed for the project. Most of them are unneeded anyways.

Basically, you should only have those two (posts and users) controllers, the routes.rb file and the 4 users and posts view files. Most of the other files are to be removed - again since they are auto-generated.